generator/generator.pro

```diff
@@ -86,3 +92,4 @@ HEADERS += \
     unpack_mwm.hpp \
     ways_merger.hpp \
     world_map_generator.hpp \
+
```

Comment:
No need in this blank line, no need in trailing '\' for the last entry in HEADERS.